### PR TITLE
2.4.7 Patch

### PR DESCRIPTION
**Pull request by jfy133**
Closes https://github.com/nf-core/eager/issues/983 and a broken conda environment

Tested by manually editing environment to bump the pygments to the minimal version as reported in teh error, and then linted with nf-core tools 1.9

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/eager/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by apeltzer — APPROVED** [review]
LGTM!

**Comment by TCLamnidis** [comment]
@aidaanva was adamant that we should also bump MultiQC to fix a division by 0 error. Is there a reason why we don't? and if we do, maybe double check that the dependencies work with it 👀

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,20 @@
...
+
+### `Added`
+
+### `Fixed`
+
+- [#983](https://github.com/nf-core/eager/issues/983) Bump `pygments` version due to incompatibility with MultiQC dependencies (♥ to @MinLuke for reporting)
+
+### `Dependencies`
+
+- `pygments`: 2.9 -> 2.14
```
Comment (TCLamnidis) [inline CHANGELOG.md:16]: Maybe also bump MQC?

**Review by TCLamnidis — APPROVED** [review]
partial approval. see my comment. I am fine either way, but if the latest MQC version fixes a bug in one of the modules, it seems like a good time to add that too.

**Pull request merged**